worker/test/ai-gateway/registry-alibaba.spec.ts

```diff
@@ -4826,5 +4832,754 @@ describe("Alibaba Registry Tests", () => {
           },
         }));
     });
+
+    describe("qwen3-coder-480b-a35b-instruct-fp8 with Canopy Wave Provider", () => {
+      it("should handle canopy wave provider", () =>
+        runGatewayTest({
+          model: "qwen3-coder-480b-a35b-instruct-fp8/canopywave",
+          expected: {
+            providers: [
+              {
+                url: "https://inference.canopywave.io/v1/chat/completions",
+                response: "success",
+                model: "qwen/qwen3-coder",
+                data: createOpenAIMockResponse("qwen/qwen3-coder"),
+                expects: canopywaveAuthExpectations,
+              },
+            ],
+            finalStatus: 200,
+          },
+        }));
+
+      it("should auto-select canopy wave provider when none specified", () =>
```

Comment:
Pls remove this test, since it may vary once we have other providers enabled for this model.